cmake/Modules/LocalAom.cmake

```diff
@@ -91,9 +91,6 @@ else()
     set(ENABLE_TESTDATA 0 CACHE INTERNAL "")
     set(ENABLE_TESTS 0 CACHE INTERNAL "")
     set(ENABLE_TOOLS 0 CACHE INTERNAL "")
-    if(NOT CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL CMAKE_SYSTEM_PROCESSOR)
```

Comment:
I wonder if this should have been `if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL CMAKE_SYSTEM_PROCESSOR)`; checking if the host matches the target system. Either way it's a little unsafe if the library or binaries are distributed to other systems which may not have the same baseline as the build host.